Sources/NIO/Codec.swift

```diff
@@ -337,6 +358,12 @@ extension ByteToMessageHandler {
         }
     }
 
+    private func tryDecodeWrites() {
+        if self.queuedWrites.count > 0 {
+            (self as! CanDequeueWrites).dequeueWrites()
```

Comment:
Why do we need this force cast?

Reply:
@Lukasa it's a precondition. We can only enter this is we are a `CanDequeueWrites`, I'll add a comment though